assessment.sql

```diff
@@ -5,19 +5,38 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
+CREATE TABLE licensees (id SERIAL PRIMARY KEY,
+  surname CHARACTER VARYING,
+  given_name CHARACTER VARYING,
+  gender CHARACTER VARYING,
+  height CHARACTER VARYING,
+  weight CHARACTER VARYING,
+  dob CHARACTER VARYING,
+  eye_color CHARACTER VARYING,
+  hair_color CHARACTER VARYING
+  );
+
+Created Database, Created Table, Bulk Loaded Data
 
 -- Question 2. Add `Bishop, Caren, f, 63, 132, 1943-09-26, Brown, Black` into the table of licensees.
 
--- your code here
+INSERT INTO licensees (surname,given_name,gender,height,weight,dob,eye_color,hair_color)
+VALUES ('Bishop', 'Caren', 'f', 63, 132, '1943-09-26', 'Brown', 'Black');
```

Comment:
This is where you would have run into trouble with your height and weight columns holding strings (`character varying`). Either it'd throw an error right here for them being of the wrong type, or it would silently turn them into strings for you to discover in the distant future.

Good INSERT statement. 1